test/recipes/03-test_fipsinstall.t

```diff
@@ -27,75 +27,76 @@ plan skip_all => "Test only supported in a fips build" if disabled("fips");
 plan tests => 12;
 
 my $infile = bldtop_file('providers', platform->dso('fips'));
+my $fipskey = $ENV{FIPSKEY} // '00';
 
 # fail if no module name
 ok(!run(app(['openssl', 'fipsinstall', '-out', 'fips.cnf', '-module',
              '-provider_name', 'fips',
-             '-macopt', 'digest:SHA256', '-macopt', 'hexkey:00',
+             '-macopt', 'digest:SHA256', '-macopt', "hexkey:$fipskey",
```

Comment:
Maybe add one more test that removes most of the options..
I am starting to think using macopt was a bad idea (and the digest should work as a default also).

Reply:
I can do that in a seperate PR if you dont want to do it.
It would probably be easier just to handle -hexkey & -digest

Reply:
Or you can push a commit to this PR if that's simpler.

Reply:
That normally ends up with someone forgetting to pull and then doing a force push :)
Will do in a seperate PR :)

Reply:
I removed all uses of -macopt and -mac_name, except for fipsinstall.